packages/sveltekit/package.json

```diff
@@ -17,7 +17,7 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@sveltejs/kit": "1.x"
+    "@sveltejs/kit": ">=1.x"
```

Comment:
To be a bit more cautious, should we make this `>= 1 <= 2`?

Reply:
We can ofc but for example, in Astro, we opted to leave an open end given that we'll need support for a new major anyway (https://github.com/getsentry/sentry-javascript/pull/9683)

Reply:
fine with me 👍

Reply:
I'd rather we explicitly opt-in to every major version, and make this `"@sveltejs/kit": "^1.0 || ^2.0"` like what we do for nextjs.

Reply:
ok, changed it to an explicit range. Don't have strong opinions here, so fine with anything 😅